tests/src/test/scala/parser/LinePositionSuite.scala

```diff
@@ -0,0 +1,68 @@
+import org.scalatest._
+import java.net._
+import java.io._
+
+import scala.meta._
+import scala.meta.dialects.Scala211
+import scala.meta.internal.{ast => impl}
+import scala.meta.tql._
+
+class LinePositionSuite extends ParseSuite {
+  var dir = new File(new File(System.getProperty("sbt.paths.tests.source")).getAbsolutePath)
+  def isProjectRoot(dir: File) = dir != null && new File(dir.getAbsolutePath + File.separatorChar + "project" + File.separatorChar + "build.scala").exists
+  while (dir != null && !isProjectRoot(dir)) dir = dir.getParentFile
+  test("ProjectDir (" + dir.getAbsolutePath + ")")(assert(isProjectRoot(dir)))
+
+  val ignoredFiles = List(
+    "build.scala",
+    "Token.scala",
+    "Api.scala"
+  )
+
+  val findAllDefn = topDown(collect {
+    case clss: impl.Defn.Class =>
```

Comment:
Usually one just calls those `x`, which obviates the need to come up with weird names :)